packages/react-components/src/components/utils/useLongPress.ts

```diff
@@ -0,0 +1,55 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { useRef, useState } from 'react';
+
+export default function useLongPress(onClick: () => void, onLongPress: () => void) {
+  const [action, setAction] = useState('');
+  const timerRef = useRef<NodeJS.Timeout>();
+  const [isLongPress, setIsLongPress] = useState(false);
+
+  function startPressTimer() {
+    setIsLongPress(false);
+    timerRef.current = setTimeout(() => {
+      setIsLongPress(true);
+      setAction('longpress');
+    }, 500);
```

Comment:
is half a second long enough?

Reply:
I played around with it, 1 second feels too long. feel free to try it out and let me know what you think :)